starknet_py/net/full_node_client.py

```diff
@@ -167,11 +167,11 @@ async def get_block_with_receipts(
         if block_identifier == {"block_id": "pending"}:
             return cast(
                 PendingStarknetBlockWithReceipts,
-                PendingStarknetBlockWithReceiptsSchema().load(res, unknown=EXCLUDE),
+                PendingStarknetBlockWithReceiptsSchema().load(res),
             )
         return cast(
             StarknetBlockWithReceipts,
-            StarknetBlockWithReceiptsSchema().load(res, unknown=EXCLUDE),
+            StarknetBlockWithReceiptsSchema().load(res),
         )
 
```

Comment:
Why `unknown=EXCLUDE` is not removed from all methods e.g. `get_block_with_txs`?

Reply:
The prime problem of this is working with devnet because in a few cases, they are inconsistent with starknet-spec.

Reply:
you mean starknet-devnet-rs :+1: ok then, the issue of pending+latest blocks was raised a couple of times with the devnet team, so I think we should just wait